### ISSUE TITLE
BatchNorm momentum in BNInception

### ISSUE THREAD

**Issue by artkorenev** [Cadene/pretrained-models.pytorch#44]
Hi,

I accidentally spotted that `momentum` value for `BatchNorm2d` in BNInception is set to `0.9` which seems to be quite extreme for batch normalization given that the default value is `0.1`
Isn't it a typo?

Thx

**Comment by Cadene** [comment]
Thanks for reporting this issue. It might be a type. Did you try to fine tune this network with different values for the batch normalization momentum? Please, let me know. Thanks!

Do not hesitate to send a pull request :)

**Comment by artkorenev** [comment]
No, I haven't used any of these models yet, I just spotted the typo completely accidentally when was reading the code

**Comment by Cadene** [comment]
Should be fixed https://github.com/Cadene/pretrained-models.pytorch/commit/87e97515ec4480844e188d82e87e06376337a3af
